backend/services/global_vars_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022, Yahoo Inc.
```

Comment:
Maybe it's worth updating the copyright?

Reply:
whoops! good catch